client/state/action-types.js

```diff
@@ -935,6 +935,10 @@ export const REWIND_BACKUP_PROGRESS_REQUEST = 'REWIND_BACKUP_PROGRESS_REQUEST';
 export const REWIND_BACKUP_REQUEST = 'REWIND_BACKUP_REQUEST';
 export const REWIND_BACKUP_UPDATE_ERROR = 'REWIND_BACKUP_UPDATE_ERROR';
 export const REWIND_BACKUP_UPDATE_PROGRESS = 'REWIND_BACKUP_UPDATE_PROGRESS';
+export const REWIND_BACKUPS_RECEIVE = 'REWIND_BACKUPS_RECEIVE';
+export const REWIND_BACKUPS_REQUEST = 'REWIND_BACKUPS_REQUEST';
+export const REWIND_BACKUPS_REQUEST_FAILURE = 'REWIND_BACKUPS_REQUEST_FAILURE';
+export const REWIND_BACKUPS_REQUEST_SUCCESS = 'REWIND_BACKUPS_REQUEST_SUCCESS';
```

Comment:
Similarly do we need Success and Failure actions?

Reply:
Yeah, not needed if we're not tracking the requesting state. Dropped them in f3275c3